UsbAccess/UsbAccess.cpp

```diff
@@ -31,7 +32,8 @@ const string WPEFramework::Plugin::UsbAccess::EVT_ON_USB_MOUNT_CHANGED = "onUSBM
 const string WPEFramework::Plugin::UsbAccess::EVT_ON_ARCHIVE_LOGS = "onArchiveLogs";
 const string WPEFramework::Plugin::UsbAccess::REGEX_BIN = "[\\w-]*\\.{0,1}[\\w-]*\\.bin";
 const string WPEFramework::Plugin::UsbAccess::REGEX_FILE =
-				"^[^.].*\\.(png|jpg|jpeg|tiff|tif|bmp|mp4|mov|avi|mp3|wav|m4a|flac|mp4|aac|wma|txt|bin|enc|ts)$";
```

Comment:
Checking why this regex is getting changed. We would allow all valid file names. fyi refer to https://github.com/rdkcentral/rdkservices/commit/f6d884c1cc2ae7cabc99d910ac02442784bbbd7a

Reply:
Can we add unit tests if its not present to test these scenarios.